pkg/apis/eventsource/v1alpha1/types.go

```diff
@@ -303,7 +303,7 @@ type SFTPEventSource struct {
 	Username *corev1.SecretKeySelector `json:"username,omitempty" protobuf:"bytes,3,opt,name=username"`
 	// Password required for authentication if any.
 	Password *corev1.SecretKeySelector `json:"password,omitempty" protobuf:"bytes,4,opt,name=password"`
-	// SSHKeySecret refers to the secret that contains SSH key
+	// SSHKeySecret refers to the secret that contains SSH key. Key needs to contain private key and public key.
```

Comment:
It changes to require both private and public keys?

Reply:
Actually the implementation required private and public key already. Only adjusted the comment to make it clear.